[15.0][MIG] l10n_es_facturae_face: Migration to 15.0

pedroguirao opened the pull request.

etobella committed:
[ADD] l10n_es_facturae_face

etobella committed:
[MIG] l10n_es_facturae_face: Migration to 11.0

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

pedrobaeza committed:
[FIX] l10n_es_facturae_face: Avoid some problems + deactivate test

They are failing and breaking branch, so this is the best soluction for now

etobella committed:
[FIX] facturae_face: Certificado en formato DER

etobella committed:
[FIX] facturae_face: Missing parent field

etobella committed:
[FIX] l10n_es_facturae_face: missing sudos

etobella committed:
[FIX] facturae_face: ejecutar como sudo debido a cambio del mimetype

Commit:
[12.0][MIG] l10n_es_facturae_face

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

etobella committed:
[12.0][IMP] l10n_es_facturae_face: Add cron to update data, removing reference to mandate

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

etobella committed:
[FIX] facturae_face: Error en la validación del correo electrónico

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
l10n_es_facturae_face 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

etobella committed:
[REF] l10n_es_facturae_face: Black python code

etobella committed:
[MIG] l10n_es_facturae_face

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

etobella committed:
[FIX] l10n_es_facturae_face: Fix tests

OCA-git-bot committed:
l10n_es_facturae_face 13.0.1.0.2

etobella committed:
[REF] l10n_es_facturae_face: Move EDI logic here

We will join e.Fact and FACe in a single module

etobella committed:
[MIG] l10n_es_facturae_face: migration to 14.0

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

pedroguirao committed:
[IMP] l10n_es_facturae_face: black, isort, prettier

pedroguirao committed:
[MIG] l10n_es_facturae_face: Migration to 15.0

etobella commented on l10n_es_facturae_face/components/edi_webservice_receive_face_l10n_es_facturae_face_update.py:
```diff
@@ -33,7 +33,9 @@ def receive(self):
             raise UserError(
-                _("Connection with FACe returned error %s - %s")
-                % (response.resultado.codigo, response.resultado.descripcion)
+                _(
```
Por que este cambio?

etobella commented on l10n_es_facturae_face/components/webservice_face.py:
```diff
@@ -78,7 +78,9 @@ def cancel(self, public_crt, private_key, identifier, motive):
             raise UserError(
-                _("Connection with FACe returned error %s - %s")
-                % (response.resultado.codigo, response.resultado.descripcion)
+                _(
```
Idem

etobella reviewed (commented):
Un par de comentarios, que no sé si funcionaran con el translate de odoo. Has podido probarlo en el entorno de pruebas de FACe?

LoisRForgeFlow committed:
[FIX] l10n_es_facturae_face: skip check step.

The backend does not have a check component, so it will raise
a NotImplementedError if we do not skip the check.

etobella commented:
Por cierto, puedes añadir este PR también #2203 ?

etobella reviewed (approved):
Testeado en local contra el entorno de desarrollo de FACe. Revisados los cambios de los commits de migración y añadidos algunos commits faltantes

pedrobaeza commented:
/ocabot migration l10n_es_facturae_face
/ocabot merge nobump
